apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -923,27 +935,59 @@ describe('B2BApp permissions for service user', () => {
         expect(countByUser3).toBe(1)
     })
 
-    // NOTE: Currently these schemes are not added to read/manage the service user.
-    //       It is used to local test the preprocessor
-    test.skip('Ticket and TicketComment', async () => {
+    test('Ticket domain', async () => {
         const [organization] = await registerNewOrganization(user)
 
         const [app] = await createTestB2BApp(support)
         await createTestB2BAppContext(support, app, organization, { status: 'Finished' })
         const [right] = await createTestB2BAppAccessRight(support, serviceUser.user, app)
 
         const [property] = await createTestProperty(user, organization)
-        const [ticket] = await createTestTicket(user, organization, property)
+        const [classifier] = await createTestTicketClassifier(support)
+        const unitName = faker.random.alphaNumeric(3)
+        const unitType = FLAT_UNIT_TYPE
+        const [contact] = await createTestContact(user, organization, property, {
+            unitName,
+            unitType,
+        })
+
+        const [ticket] = await createTestTicket(user, organization, property, {
+            contact: { connect: { id: contact.id } },
+            assignee: { connect: { id: user.user.id } },
+            executor: { connect: { id: user.user.id } },
+            classifier: { connect: { id: classifier.id } },
+            unitName,
+            unitType,
+        })
 
         const [ticketComment] = await createTestTicketComment(user, ticket, user.user)
+        const [ticketFile] = await createTestTicketFile(user, ticket)
+        const [ticketCommentFile] = await createTestTicketCommentFile(user, ticket, ticketComment)
 
         // B2BApp without permissions
-        const countWithoutPermissions = await TicketComment.count(serviceUser, {})
-        expect(countWithoutPermissions).toBe(0)
-        const contactWithoutPermissions = await TicketComment.getOne(serviceUser, {
-            ticket: { id: ticket.id },
+        await expectToThrowAccessDeniedErrorToCount(async () => {
+            await Ticket.count(serviceUser, {})
+        })
+        await expectToThrowAccessDeniedErrorToObjects(async () => {
+            await Ticket.getOne(serviceUser, {
+                id: ticket.id,
+            })
+        })
+        await expectToThrowAccessDeniedErrorToObjects(async () => {
+            await Ticket.getAll(serviceUser, {})
+        })
+
+        await expectToThrowAccessDeniedErrorToCount(async () => {
+            await TicketComment.count(serviceUser, {})
+        })
+        await expectToThrowAccessDeniedErrorToObjects(async () => {
+            await TicketComment.getOne(serviceUser, {
+                ticket: { id: ticket.id },
+            })
+        })
+        await expectToThrowAccessDeniedErrorToObjects(async () => {
+            await TicketComment.getAll(serviceUser, {})
```

Comment:
What about tests for `TicketFile.get...` and `TicketCommentFile.get...`?

Reply:
they exists